dev-libs/boehm_gc/boehm_gc-7.6.0.recipe

```diff
@@ -10,8 +10,8 @@ COPYRIGHT="1988, 1989 Hans-J. Boehm, Alan J. Demers
 	1991-1996 Xerox Corporation.
 	1996-1999 Silicon Graphics.
 	1999-2011 Hewlett-Packard Development Company, L.P."
-LICENSE="BOEHM"
-REVISION="2"
+LICENSE="Boehm"
+REVISION="3"
 SOURCE_URI="http://www.hboehm.info/gc/gc_source/gc-$portVersion.tar.gz"
```

Comment:
Shall we use this variant?
```
SOURCE_URI="https://github.com/ivmai/bdwgc/releases/download/v$portVersion/gc-$portVersion.tar.gz
	http://www.hboehm.info/gc/gc_source/gc-$portVersion.tar.gz"
```
Advantages:
1) I uses https instead of http.
2) If GitHub is down, haikuporter will fallback to hboehm.info.

(Reminder: the 80 chars limit does not apply to `SOURCE_URI`.)

Reply:
> SOURCE_URI="https://github.com/ivmai/bdwgc/releases/download/v$portVersion/gc-$portVersion.tar.gz
>	http://www.hboehm.info/gc/gc_source/gc-$portVersion.tar.gz"

Done